core/blockchain.go

```diff
@@ -1411,15 +1413,17 @@ func (bc *BlockChain) writeBlockWithoutState(block *types.Block, td *big.Int) (e
 	return nil
 }
 
-// writeKnownBlock updates the head block flag with a known block
+// writeKnownBlockAsHead updates the head block flag with a known block
```

Comment:
This is just a renaming because it's a better name for the method. Not impactful on meaningful logic.